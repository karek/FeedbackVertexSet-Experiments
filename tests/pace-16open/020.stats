74,92,9
