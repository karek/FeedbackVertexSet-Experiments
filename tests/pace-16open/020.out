8
15 61 14 50 62 57 51 19 
